### ISSUE TITLE
[Designer] Text is not visible in Dark theme 

### ISSUE THREAD

**Issue by rvinothrajendran** [microsoft/AdaptiveCards#5270]
**Steps** 

Browser :  Microsoft Edge  &  Version 87.0.664.75 

1. https://adaptivecards.io/designer/ 
2. Select the host app in Dark Theme 
3. click the new card option 

**the button text is not visible** 

![image](https://user-images.githubusercontent.com/16264167/104486875-a54e6600-55cc-11eb-862e-c0253dd35ab2.png)

**Comment by shalinijoshi19** [comment]
@dclaux known at all?

**Comment by jonmill** [comment]
@dclaux - is this still an issue or has this been fixed already?

**Comment by paulcam206** [comment]
Addressing this issue is not currently on our roadmap, so closing for now. We may return to this in the future. Feel free to continue the conversation in this issue.
